specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.2-preview.1/TextAnalytics.json

```diff
@@ -0,0 +1,2669 @@
+{
```

Comment:
When I do a generation, autorest shows these warnings:
```
WARNING (PreCheck/SchemaMissingType): The schema 'TasksState-tasks-entityRecognitionTasksItem' with an undefined type and 'allOf'/'anyOf'/'oneOf' is a bit ambigious. This has been auto-corrected to 'type:object'

WARNING (PreCheck/SchemaMissingType): The schema 'TasksState-tasks-entityRecognitionPiiTasksItem' with an undefined type and 'allOf'/'anyOf'/'oneOf' is a bit ambigious. This has been auto-corrected to 'type:object'

WARNING (PreCheck/SchemaMissingType): The schema 'TasksState-tasks-entityLinkingTasksItem' with an undefined type and 'allOf'/'anyOf'/'oneOf' is a bit ambigious. This has been auto-corrected to 'type:object'

WARNING (PreCheck/SchemaMissingType): The schema 'TasksState-tasks-keyPhraseExtractionTasksItem' with an undefined type and 'allOf'/'anyOf'/'oneOf' is a bit ambigious. This has been auto-corrected to 'type:object'

WARNING (PreCheck/SchemaMissingType): The schema 'TasksState-tasks-sentimentAnalysisTasksItem' with an undefined type and 'allOf'/'anyOf'/'oneOf' is a bit ambigious. This has been auto-corrected to 'type:object'

WARNING (PreCheck/SchemaMissingType): The schema 'TasksState-tasks-customClassificationTasksItem' with an undefined type and 'allOf'/'anyOf'/'oneOf' is a bit ambigious. This has been auto-corrected to 'type:object'

WARNING (PreCheck/SchemaMissingType): The schema 'TasksState-tasks-customEntityRecognitionTasksItem' with an undefined type and 'allOf'/'anyOf'/'oneOf' is a bit ambigious. This has been auto-corrected to 'type:object'

WARNING (PreCheck/SchemaMissingType): The schema 'HealthcareEntity' with an undefined type and decalared properties is a bit ambigious. This has been auto-corrected to 'type:object'

WARNING (PreCheck/CheckDuplicateSchemas): Checking for duplicate schemas, this could take a (long) while.  Run with --verbose for more detail.
```

Reply:
I have fixed the warnings and is only showing the last one which is 

```
WARNING (PreCheck/CheckDuplicateSchemas): Checking for duplicate schemas, this could take a (long) while.  Run with --verbose for more detail.
```

Let me know if you know how to fix it?

@maririos @deyaaeldeen

Reply:
@suhas92 thanks! I retried and now see this one:
```
WARNING (PreCheck/SchemaMissingType): The schema 'HealthcareEntity' with an undefined type and decalared properties is a bit ambigious. This has been auto-corrected to 'type:object'
```

Reply:
Try one more time. @deyaaeldeen

Reply:
perfect, thanks!